builtins/zlib/crc32.c

```diff
@@ -201,7 +239,7 @@ const z_crc_t FAR * ZEXPORT get_crc_table()
 #define DO8 DO1; DO1; DO1; DO1; DO1; DO1; DO1; DO1
 
 /* ========================================================================= */
-unsigned long ZEXPORT crc32(crc, buf, len)
+local unsigned long crc32_generic(crc, buf, len)
```

Comment:
In most files "local" was replaced by "static", where-as here it is added.  Is that an over-sight or intentional?

Reply:
@pcanal  in crc32, there is introduced a trick https://github.com/oshadura/root/blob/d7eef26043e57146b3fd86d1394045c3d555c773/builtins/zlib/crc32.c#L71,  so local here is not really "local" :) I think it is done to have easier rebase later on zlib/madler, it is not my original code, it is inherited from zlib/cloudflare.

Reply:
Do we still define 'local' somewhere? (I thought I saw its definition being removed)

Reply:
```local``` is everywhere, but redefined as  ```#define local static```. I checked cms-externals/cloudflare/gcc.amd64 and it is left there in the same way. The reason is probably to be able to rebase on original zlib. Do you prefer to change me this trick?